packages/story-editor/src/components/panels/design/layer/layer.js

```diff
@@ -267,11 +268,11 @@ const LayerAction = styled(Button).attrs({
   padding: 0;
 
   /*
-   * all of our Icons right now have an embedded padding,
-   * however the new designs just disregard this embedded
-   * padding, so to accommodate, we'll make the icon its
-   * intended size and manually center it within the button.
-   */
+    * all of our Icons right now have an embedded padding,
+    * however the new designs just disregard this embedded
+    * padding, so to accommodate, we'll make the icon its
+    * intended size and manually center it within the button.
+    */
```

Comment:
This doesn't look right? We don't have any formatter rules for this in place, but I don't think this indent should be there - it looks off?

<img width="503" alt="Screen Shot 2022-06-08 at 14 00 08" src="https://user-images.githubusercontent.com/637548/172611283-eff9b47d-316c-485d-b209-f92e8ca95f27.png">